### DISCUSSION TITLE
Logbook salvage can spawn in locked cabinets

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#14017] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Some of the new wrecks (Barsuk, R-29, Remora, and Typhon 2) have secure steel cabinets with the `locker` tag, which means that logbooks from wreck salvage missions can spawn in them. It's not uncommon for there not to be ID cards available to open these cabinets (either no captain/security corpses spawn, or a monster eats those corpses) and so the missions are impossible to complete.

Maybe there should be guaranteed ID cards that can open the cabinets (similar to the guaranteed card in Wreck1), oor just remove the `locker` tag from these cabinets so that the logbooks can't spawn.

### Reproduction steps

1. Load into a wreck salvage mission
2. Check if logbook is inside secure locker
3. Check if there are no ID cards that can open it
4. repeat steps 1-3 until you find an impossible to access logbook

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by mrwallace888** [comment]
Can confirm, we got a Barsuk mission on our run and none of the ID cards could open the cabinet.

**Comment by Regalis11** [comment]
This has been addressed in the 2024 autumn update, looks like we just forgot to close the report. :)
